src/Poseidon/Package.hs

```diff
@@ -86,10 +99,37 @@ instance ToJSON PoseidonPackage where
         "title" .= posPacTitle x,
         "description" .= posPacDescription x,
         "contributor" .= posPacContributor x,
+        "packageVersion" .= posPacPackageVersion x,
         "lastModified" .= posPacLastModified x,
         "bibFile" .= posPacBibFile x,
         "genotypeData" .= posPacGenotypeData x,
-        "jannoFile" .= posPacJannoFile x
+        "jannoFile" .= posPacJannoFile x,
+        "checksums" .= posPacChecksumList x
+        ]
+
+instance ToPrettyYaml PoseidonPackage where
```

Comment:
As far as I understand, we _can_ and should use the `ToPrettyYaml` type class to define field orders in all nested types within this Yaml structure... so also within `ContributorSpec` and `GenotypeSpec`